CommonMark/CommonMarkException.cs

```diff
@@ -8,7 +8,7 @@ namespace CommonMark
     /// <summary>
     /// An exception that is caught during CommonMark parsing or formatting.
     /// </summary>
-#if v2_0 || v3_5 || v4_0 || v4_5
+#if v2_0 || v3_5 || v4_0 || (v4_5 && !portable)
```

Comment:
How is this different functionality? Serialization should work the same,
it's just that the attribute is unavailable in the specific profile.
On Nov 24, 2015 12:41, "Kārlis Gaņģis" <notifications@github.com> wrote:

> In CommonMark/CommonMarkException.cs
> <https://github.com/Knagis/CommonMark.NET/pull/56#discussion_r45719454>:
>
> > @@ -8,7 +8,7 @@ namespace CommonMark
> >      /// <summary>
> >      /// An exception that is caught during CommonMark parsing or formatting.
> >      /// </summary>
> > -#if v2_0 || v3_5 || v4_0 || v4_5
> > +#if v2_0 || v3_5 || v4_0 || (v4_5 && !portable)
>
> I don't like that here the portable 4.5 will differ in functionality from
> the separate 4.5 - they should be the same
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Knagis/CommonMark.NET/pull/56/files#r45719454>.
>